metricbeat/module/docker/event/event_integration_test.go

```diff
@@ -0,0 +1,71 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// +build integration
+
+package event
+
+import (
+	"testing"
+	"time"
+
+	"github.com/elastic/beats/auditbeat/core"
+	"github.com/elastic/beats/libbeat/tests/compose"
+	"github.com/elastic/beats/metricbeat/mb"
+	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
+)
+
+func TestData(t *testing.T) {
+	ms := mbtest.NewPushMetricSetV2(t, getConfig())
+	var events []mb.Event
+	done := make(chan interface{})
+	go func() {
+		events = mbtest.RunPushMetricSetV2(10*time.Second, 1, ms)
+		close(done)
+	}()
+
+	compose.EnsureUp(t, "apache")
```

Comment:
Is an event also generated if `apache` is up? Could we stop the container as cleanup?

Or as another option, does creating networks or volumes also generate events? Maybe we could create a dummy network or volume, it could be created with a random name and deleted with defer.

Reply:
I used the API to create a cheaper event, I still want to keep a container create event because it shows a richer example

Reply:
Oh, great, could we also delete the container so they don't stay there after consecutive calls?

Reply:
I've pushed code to remove it :+1: